src/evaluation_system/tests/esgf_test.py

```diff
@@ -189,8 +190,8 @@ def test_freva_esgf_method(dummy_config):
 
 def test_find_files(capsys, search_dict, dummy_config):
     result_to_be = [
-        "output1/MPI-M/MPI-ESM-LR/decadal2000/mon/atmos/Amon/r1i1p1/tas/1/"
-        "tas_Amon_MPI-ESM-LR_decadal2000_r1i1p1_200101-201012.nc",
+        # "output1/MPI-M/MPI-ESM-LR/decadal2000/mon/atmos/Amon/r1i1p1/tas/1/"
+        # "tas_Amon_MPI-ESM-LR_decadal2000_r1i1p1_200101-201012.nc",
```

Comment:
Are these gone from ESGF? Can you double check with the maintainers?

Reply:
hmm, who i need to check with him? Let me rerun the ci on working days, it might be fixed ;)

Reply:
As I suspected, the ESGF test-related data is only available on weekdays. Beautiful!